### PR TITLE
No gotham

### PR DESCRIPTION
**Pull request by OriPekelman**

### CODE REVIEW

**Comment by GuGuss** [comment]
Hi @OriPekelman, this needs a rebase.

**Comment by OriPekelman** [comment]
ouais ouais, this can wait

**Comment by sruggier** [comment]
Sorry, my mistake, I didn't mean to close this.

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <link rel="stylesheet" href="/gitbook/style.css">
   <link rel="stylesheet" href="/gitbook/plugins/gitbook-theme-platform/platform.css">
   <!-- head:end -->
-  <link rel='stylesheet' type='text/css' href='//cloud.typography.com/7640312/659566/css/fonts.css'>
+<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
```
Comment (kotnik) [inline index.html]: HTTPS, please.

**Comment by Crell** [comment]
@OriPekelman Is this something you can return to, or at least rebase so we can continue it?  If it's no longer relevant, please close.

**Pull request closed without merging**